pipelined/bevy_sprite2/src/texture_atlas.rs

```diff
@@ -22,30 +22,19 @@ pub struct TextureAtlas {
 
 #[derive(Debug, Clone, TypeUuid, Reflect)]
 #[uuid = "7233c597-ccfa-411f-bd59-9af349432ada"]
-pub struct TextureAtlasSprite {
-    pub color: Color,
```

Comment:
You said in the PR description that the implementation for sprite color was incomplete, and that's why you removed it. Do you mean that, previously, this field didn't do anything at all?

Just wanna make sure nothing is being regressed.

Reply:
Yap, it wasn't used anywhere.